apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,247 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type CourseRequestRow } from './instructorRequestCourse';
+
+export const RequestCourse = ({
```

Comment:
Just use a regular function rather than an arrow function.
```suggestion
export function RequestCourse({
```

Reply:
Changed in 9664c3b